src/Magnum/BulletIntegration/MotionState.h

```diff
@@ -69,25 +69,27 @@ only update the motion state of non-static objects and while
 `btDynamicsWorld::stepSimulation()` is called.
 
 */
-class MAGNUM_BULLETINTEGRATION_EXPORT MotionState: public SceneGraph::AbstractBasicFeature3D<btScalar>, private btMotionState {
+class MotionState: public SceneGraph::AbstractBasicFeature3D<btScalar>, private btMotionState {
     public:
         /**
          * @brief Constructor
          * @param object    Object this motion state belongs to
          */
         template<class T> MotionState(T& object);
 
+        ~MotionState() = default;
+
         /** @brief Motion state */
         btMotionState& btMotionState() { return *this; }
 
     private:
-        void MAGNUM_BULLETINTEGRATION_LOCAL getWorldTransform(btTransform& worldTrans) const override;
-        void MAGNUM_BULLETINTEGRATION_LOCAL setWorldTransform(const btTransform& worldTrans) override;
+        void MAGNUM_BULLETINTEGRATION_EXPORT getWorldTransform(btTransform& worldTrans) const override;
+        void MAGNUM_BULLETINTEGRATION_EXPORT setWorldTransform(const btTransform& worldTrans) override;
```

Comment:
I saw the OSX CI fail on some linker error. I guess this is probably related to the above changes of the EXPORT/LOCAL macros. I think reverting those and defaulting the destructor inside the `*.cpp` file (instead of having it inline in the header) might help.

Overriden virtuals don't need to be exported (unless you are calling them directly, which you can't, because they are private).